src/Assert.php

```diff
@@ -2099,6 +2099,10 @@ protected static function valueToString($value)
                 return \get_class($value).': '.self::valueToString($value->__toString());
             }
 
+            if ($value instanceof \DateTime || $value instanceof \DateTimeImmutable) {
```

Comment:
Classes are imported in this package. Could you please add a use statement and make this a non FQN?

For reffernce, functions are not, as that isn't possible in php 5.3